tests/unit-tests/chaos-experiments/models/chaosExperimentsManager-test.js

```diff
@@ -6,6 +6,7 @@ const rewire = require('rewire');
 
 const database = require('../../../../src/chaos-experiments/models/database/databaseConnector');
 const manager = rewire('../../../../src/chaos-experiments/models/chaosExperimentsManager');
```

Comment:
fyi - we don't use rewire functionality here - (__set__ )
I prefer to remove it as this is unhealthy methodology in testing